### PR TITLE
Remove unnecessary test prints

### LINKED ISSUE #1240: Remove useless prints from tests
## 🚀 Feature

Some tests contain `print`s that probably were used for debugging and were not removed. It is better to remove them.

For example, 
https://github.com/pytorch/ignite/blob/44c60564bf59a3028ab89091cbd3d73def8be86f/tests/ignite/contrib/engines/test_common.py#L71

https://github.com/pytorch/ignite/blob/44c60564bf59a3028ab89091cbd3d73def8be86f/tests/ignite/contrib/engines/test_common.py#L244

https://github.com/pytorch/ignite/blob/44c60564bf59a3028ab89091cbd3d73def8be86f/tests/ignite/contrib/handlers/test_param_scheduler.py#L631

etc

### PR DESCRIPTION
**Pull request by nzare**
Fixes #1240 

Remove useless prints from tests

### CODE REVIEW

File: tests/ignite/distributed/utils/test_serial.py [hunk 1]
```diff
@@ -9,7 +9,6 @@ def test_no_distrib(capsys):
 
     from ignite.distributed.utils import _model
 
-    print("test_no_distrib : dist: ", dist.is_available())
```
Comment (vfdev-5) [inline tests/ignite/distributed/utils/test_serial.py:12]: Could you please remove both prints and "from ignite.distributed.utils import _model" if it is not used below.

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR.
I left a comment in the code.
Could you please also remove : 
- https://github.com/pytorch/ignite/blob/master/tests/ignite/engine/test_engine.py#L620
- https://github.com/pytorch/ignite/blob/master/tests/ignite/distributed/comp_models/test_xla.py#L189-L191
- https://github.com/pytorch/ignite/blob/master/tests/ignite/metrics/test_top_k_categorical_accuracy.py#L71

**Comment by nzare** [comment]
Can you please review it now ? 
Thank You

**Comment by vfdev-5** [comment]
@nzare could you please fix flake8 error : https://travis-ci.org/github/pytorch/ignite/jobs/726552697#L242

**Comment by nzare** [comment]
I checked flake8 error
./tests/ignite/metrics/test_top_k_categorical_accuracy.py:70:1: W293 blank line contains whitespace

I tried fixing with autopep and tried changing spacing etc but none of them seems to work

**Comment by vfdev-5** [comment]
you can try to run `black .` on the codebase. For details, please read
CONTRIBUTING.md

**Comment by vfdev-5** [comment]
@nzare could you please merge master branch to this PR as it modifies back a recent commit. Thanks

**Comment by nzare** [comment]
Merged master and opened a new pull request as it was facing some merging issue. 
#1292

**Pull request closed without merging**